src/main/kotlin/org/opensearch/commons/alerting/model/Monitor.kt

```diff
@@ -43,6 +43,7 @@ data class Monitor(
     val uiMetadata: Map<String, Any>,
     val dataSources: DataSources = DataSources(),
     val deleteQueryIndexInEveryRun: Boolean? = false,
+    val shouldPersistFindingsAndAlerts: Boolean? = false,
```

Comment:
shouldn't this be true by default?

Reply:
the field is named now as `shouldCreateSingleAlertForFindings`. & its set to `False` by default.